### ISSUE TITLE
ADE20k Learning Rate

### ISSUE THREAD

**Issue by cjrd** [facebookresearch/mae#52]
Section A.4 of the paper has "We search for the optimal lr for each entry in Table 5 " when referring to segmentation on ADE20k. Could you share these learning rates? My reproduced baselines are about 2 points too low.

Thanks!

**Comment by implus** [comment]
see our reproduction here: https://github.com/facebookresearch/mae/issues/2#issuecomment-1028552892

**Comment by endernewton** [comment]
Closing with the reproduced results above. Please check lr, layer-wise rate decay, and drop path rate similar to what we have done in detection.

**Comment by youngwanLEE** [comment]
@endernewton Hi,

Are LR, layer-wise rate decay, and drop-path hyper-parameters of UperNet the same as those of detection?

Do you have a plan to release the segmentation code and weights?

**Comment by hellojialee** [comment]
Dose layer-wise rate decay really make differences obviously?
